### DISCUSSION TITLE
Add some FOSS content

### DISCUSSION THREAD

**Discussion by IronSim** [geraldohomero/best-foss-alternatives#45] [Ideas]
Thank you very much for this awesome list.

i found 3 more useful FOSS software which i think, might fit well to be added here. I tried to contribute, but i'm very new. So i try it this way (sorry if this is wrong)

## Authenticators
[Aegis](https://getaegis.app/) - <Android>
## Image editors
[Darktable](https://www.darktable.org/)
## VPN
[Wireguard](https://www.wireguard.com/)

**Comment by geraldohomero** [comment]
Added here

[Commit](https://github.com/geraldohomero/best-foss-alternatives/commit/c8a186e98b07c41b6b99ca4337d12fdabf65fdc7)

**Reply by geraldohomero** [reply]
If you want to contribute, please you can read https://github.com/geraldohomero/best-foss-alternatives/blob/main/CONTRIBUTING.md

Or you can just add a new [issue](https://github.com/geraldohomero/best-foss-alternatives/issues) with your request
